### PR TITLE
EDU-1197 media library administration tab

### PR DESCRIPTION
**Pull request by ahelmberger**
Closes https://educandu.atlassian.net/browse/EDU-1197

### CODE REVIEW

**Comment by ahelmberger** [comment]
Again I'm sorry for the number of affected files, but in order to make the "redaction" page somewhat streamlined, I chose to also adjust the documents tab as well.

File: src/services/saml-config-service.js [hunk 1]
```diff
@@ -71,7 +71,7 @@ export default class SamlConfigService {
         this._identityProviderMap.set(provider.key, {
           ...provider,
           resolvedMetadata,
-          lastUpdatedOn: new Date()
+          lastUpdateOn: new Date()
```
Comment (adrianaluchian) [inline src/services/saml-config-service.js:74]: Oh we had a typo here? 😳

**Comment by adrianaluchian** [comment]
Documents tab is much more useful now! 😍 
And good that you streamlined the table 👏

**Review by adrianaluchian — APPROVED** [review]

File: src/services/saml-config-service.js (hunk 1 above)
Comment (ahelmberger) [reply src/services/saml-config-service.js]: :scream: That happened accidenatlly. Thanks for pointing it out. I reverted it!

Comment (adrianaluchian) [reply src/services/saml-config-service.js]: 👌

**Pull request merged**